v3/src/components/graph/adornments/univariate-measures/plotted-value/plotted-value-adornment-model.ts

```diff
@@ -9,7 +9,8 @@ export const PlottedValueAdornmentModel = UnivariateMeasureAdornmentModel
   .props({
     type: types.optional(types.literal(kPlottedValueType), kPlottedValueType),
     formula: types.optional(Formula, () => Formula.create()),
-    labelTitle: types.optional(types.literal(kPlottedValueValueTitleKey), kPlottedValueValueTitleKey)
+    labelTitle: types.optional(types.literal(kPlottedValueValueTitleKey), kPlottedValueValueTitleKey),
+    error: types.optional(types.string, "")
```

Comment:
In this case, the simpler form should suffice:
```suggestion
    error: ""
```